scripts/nft/pullAllCompressedNfts.ts

```diff
@@ -0,0 +1,53 @@
+import axios from "axios";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+async function fetchCompressedNfts(walletAddress: string): Promise<any[]> {
+  const url: string | undefined = process.env.RPC_URL;
```

Comment:
Add env variables checking at the start of the script. Refer to other files to see how I've done it.